frontend/src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React, {FunctionComponent, ReactElement} from "react";
+import SerokellLogo from "~/assets/serokell.svg"
+
+export const App: FunctionComponent = (): ReactElement => {
+    // TODO remove. Code needed to check eslint react config
```

Comment:
Remove when?

Reply:
well, I can remove it now, I just wanted Sasha to import in it Idea and make sure Idea highlights this, then remove. It's just crucial to have this warning turned on IDE level because sometimes you forgot about some React invariants and misuse their API but their lint rules in IDE will notify you about it.